packages/wonder-stuff-testing/src/jest/wait.ts

```diff
@@ -1,8 +1,5 @@
 import {verifyRealTimers} from "./internal/verify-real-timers";
 import {unverifiedWait} from "./internal/unverified-wait";
-import {assertJest} from "./internal/assert-jest";
-
-assertJest();
```

Comment:
**note:** The `wait` call invokes a `verifyRealTimers` call, which in turn will call `assertJest` so we're still covered, just not at import time.